fvm/transactionInvocator.go

```diff
@@ -163,6 +163,9 @@ func (i *TransactionInvocator) Process(
 		txError = NewTransactionStorageLimiter().Process(vm, ctx, proc, sth, programs)
 	}
 
+	proc.Logs = append(proc.Logs, env.getLogs()...)
```

Comment:
Are we sure we want to drop logs if tx fails? Aren't they useful for debugging?

Reply:
I thought the same thing, that's why I moved it to the above error check step. for transactions, this will be disabled anyway.